documentation/asciidoc/computers/camera/libcamera_differences.adoc

```diff
@@ -1,42 +1,42 @@
-=== Differences compared to `Raspicam` apps
+=== Differences between `rpicam` and `raspicam`
 
-Whilst the `rpicam-apps` attempt to emulate most features of the legacy `Raspicam` applications, there are some differences. 
+The `rpicam-apps` emulate most features of the legacy `raspicam` applications. However, users may notice the following differences:
 
-* The use of Boost `program_options` doesn't allow multi-character short versions of options, so where these were present they have had to be dropped. The long form options are named the same way, and any single-character short forms are preserved.
+* Boost `program_options` don't allow multi-character short versions of options, so where these were present they have had to be dropped. The long form options are named the same way, and any single-character short forms are preserved.
 
-* `rpicam-still` and `rpicam-jpeg` do not show the capture image in the preview window.
+* `rpicam-still` and `rpicam-jpeg` do not show the captured image in the preview window.
 
-* `libcamera` performs its own camera mode selection, so the `--mode` option is not supported. It deduces camera modes from the resolutions requested. There is still work ongoing in this area.
+* `rpicam-apps` removed the following `raspicam` features:
++
+** opacity (`--opacity`)
+** image effects (`--imxfx`)
+** colour effects (`--colfx`)
+** annotation (`--annotate`, `--annotateex`)
+** dynamic range compression, or DRC (`--drc`)
+** stereo (`--stereo`, `--decimate` and `--3dswap`)
+** image stabilisation (`--vstab`)
+** demo modes (`--demo`)
++
+xref:camera_software.adoc#post-processing[Post-processing] replaced many of these features.
 
-* The following features of the legacy apps are not supported, as the code has to run on the Arm processor now. A number of these effects are now provided by the xref:camera_software.adoc#post-processing[post-processing] mechanism.
-  - opacity (`--opacity`)
-  - image effects (`--imxfx`)
-  - colour effects (`--colfx`)
-  - annotation (`--annotate`, `--annotateex`)
-  - dynamic range compression, or DRC (`--drc`)
+* `rpicam-apps` removed xref:camera_software.adoc#rotation[`rotation`] option support for 90° and 270° rotations. You can still achieve these rotations with xref:camera_software.adoc#hflip[`hflip`] and xref:camera_software.adoc#vflip[`vflip`].
```

Comment:
Is this right? I don't see how you can use hflip and vflip to create a 90-degree rotation? :upside_down_face:

Reply:
D'oh.